browser/js/products/userProducts/userProducts.js

```diff
@@ -1,12 +1,34 @@
+/**
+ * Using this as site of products put in cart
+ * This would apply to authenticated user
+ * and unauthenticated user
+ *  If authenticated user, grab from server
+ *  else use localStorage
+ *  - Victor
+ */
 app.config($stateProvider => {
 
     $stateProvider.state('products.user', {
-        url: '/products/user/:name',
-        templateUrl: 'js/userProducts/userProducts.html',
-        controller: 'UserProductCtrl'
+      url: '/products/user/:name',
+      templateUrl: 'js/products/userProducts/userProducts.html',
+      controller: 'UserProductCtrl',
+      resolve : {
+        // we don't have a backend cart yet
```

Comment:
I think we decided that we will just store it as an order with a status of 'incomplete' or something like that. Well talk about that after reacto.